ADD translations

Dominik2809 opened the pull request:
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested

@plentymarkets/ceres-io

Dominik2809 committed:
ADD translations

felixdausch reviewed (commented):
@fmutschler Bitte die Texte nochmals prüfen

fmutschler committed:
Update Template.properties

basketItemsRemovedForLanguage wurde in (en) hinsichtlich Reihenfolge und Kausalität angepasst; 
"Items have been removed from the shopping cart because they are not available in the selected language."

Dominik2809 committed:
Merge branch 'beta' into fix/remove_basket_items_without_lang

Dominik2809 committed:
UPDATE  logic for pushing notifications by error code

Dominik2809 committed:
Merge branch 'beta' into fix/remove_basket_items_without_lang

lks-mtzn commented on resources/lang/en/Template.properties:
```diff
@@ -100,6 +100,7 @@ basketCoupon = "Coupon"
 basketGross = "Gross"
 basketItemAdded = "The item was placed in the shopping cart."
 basketItemNotAdded = "The item could not be added."
+basketItemsRemovedForLanguage = "Items have been removed from the shopping cart because they are not available in the selected language."
```
Die Texte müssten noch mal an die neue Property Struktur angepasst werden.

dbohla commented:
@Dominik2809 Kann der PR nicht geschlossen werden?
